conf/full.json

```diff
@@ -0,0 +1,44 @@
+{
+  "storage": "./storage",
```

Comment:
I think this should be removed completely, as you can see https://github.com/verdaccio/verdaccio/blob/master/conf/full.yaml it's empty due I was used as reference and force us to maintain multiple config example files.